docs/index.rst

```diff
@@ -38,8 +38,7 @@ NVIDIA DALI Documentation
 
    pipeline
    data_types
-   supported_ops
```

Comment:
Should we move supported_ops into data manipulation?

Reply:
That's what I'd like to discuss.